### DISCUSSION TITLE
Can not select "Require new users to enter a reason to join"

### DISCUSSION THREAD

**Discussion by LettError** [mastodon/mastodon#20399] [Q&A]
I would like to set "Require new users to enter a reason to join" in `admin/Preferences/Administration/Site settings/`

I have set a **Closed registration message**, and settings are saved. 

But the checkbox is grayed out. How do I proceed? Thank you!

**Comment by mjankowski** [comment]
This box is only selectable when the server is in "approval required" mode.
